monai/handlers/metrics_saver.py

```diff
@@ -45,8 +45,9 @@ class MetricsSaver:
             "*" - save all the existing metric_details in `engine.state.metric_details` dict into separate files.
             list of strings - specify the metric_details of expected metrics to save.
             if not None, every metric_details array will save a separate `{metric name}_raw.csv` file.
-        batch_transform: callable function to extract the meta_dict from input batch data if saving metric details.
-            used to extract filenames from input dict data.
+        batch_transform: a callable that is used to extract the `meta_data` dictionary of
```

Comment:
my suggestion is:
batch_transform: a callable that is used to transform the output batch achieved from `ignite.engine.state.batch` to extract the `meta_data` dictionary from the input images. The purpose of extracting the `meta_data` dictionary is to get the filenames of the input images in order to store with metric details together.

Reply:
updated in the latest commit.
Thanks.